sources/packages/web/src/components/common/IdleTimeChecker.vue

```diff
@@ -60,11 +57,26 @@ export default defineComponent({
     const LAST_ACTIVITY_TIME_LOCAL_STORAGE_ITEM = "lastActivityTime";
     let countdownInterval = 0;
     let idleTimeInSeconds = 0;
+    let MAXIMUM_IDLE_TIME_FOR_WARNING_STUDENT,
+      MAXIMUM_IDLE_TIME_FOR_WARNING_INSTITUTION,
+      MAXIMUM_IDLE_TIME_FOR_WARNING_SUPPORTING_USER,
+      MAXIMUM_IDLE_TIME_FOR_WARNING_AEST;
```

Comment:
These variables are considered `any` declaration and VS Code also emits a waring. This declaration type should be avoided. Please ensure the VS Code is showing the warning below.
![image](https://github.com/user-attachments/assets/fa95f673-66c3-4d51-bb34-f5418a9130d3)